geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -96,6 +96,31 @@
     <!-- /geomesa.batchwriter -->
 
     <property>
+        <name>geomesa.mapreduce.split.count.absolute</name>
```

Comment:
maybe instead of it being .absolute it should be .max since if the value is less than the absolute we wouldn't want to create that many splits...maybe rename it to "geomesa.mapreduce.splits.max"